src/explorer/model/ITreeNode.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { ITreeItem } from "./ITreeItem";
+
+export interface ITreeNode {
+    children: ITreeNode[];
+    parent?: ITreeNode | ITreeItem | undefined;
```

Comment:
why requrie `ITreeItem`

Reply:
for root dependencies, their parent is `DependenciesMenu`, so add `ITreeItem`

Reply:
Can it be `undefined`? Is there any use case where you need to access corresponding `DependenciesMenu`?